scripts/build-installer-win64.bat

```diff
@@ -0,0 +1,102 @@
+rem DEPENDENCIES:
+rem   1. Node.js ('npm' binary in PATH)
+rem   2. 7zip    ('7z'  binary in PATH)
+
+set DEFAULT_CARDANO_BRANCH=cardano-sl-0.4
```

Comment:
Usually we want the Cardano dependency branch to match the current Daedalus branch. This is good, but you might consider setting the default branch that way.

Currently the AppVeyor script doesn't pass the 2nd argument, so it relies on this default.